Library/Formula/node.rb

```diff
@@ -25,6 +25,7 @@ class Node < Formula
   option "without-completion", "npm bash completion will not be installed"
 
   depends_on :python => :build
+  depends_on :xcode => :build
```

Comment:
Their bundled gyp contains the CLT detection patch I contributed to gyp upstream: https://github.com/npm/npm/blob/master/node_modules/node-gyp/gyp/pylib/gyp/xcode_emulation.py#L879-L898

So CLT shouldn't be an issue there. Historically it was a problem though.